### PR TITLE
Fix the bug with camera and appearance pictures

### LINKED ISSUE #48: New appearance of Player doesn't work with Camera
appearancePicture of an object has undetermined center.

### PR DESCRIPTION
**Pull request by dm-gh**
Resolves #48

### CODE REVIEW

File: src/Visual/Renderer.hs [hunk 1]
```diff
@@ -44,3 +42,14 @@ positionPicture camera pic = Scale ratio ratio (Translate x y pic)
         offset = unwrap . invertVector . positionToVector . cameraPosition $ camera
         x = fst offset
         y = snd offset
+
+
+
+objectToPicture :: Object -> Picture
+objectToPicture obj = translate' obj $ scale' obj $ appearancePicture . objectAppearance $ obj
+  where scale' = \o -> uncurry Scale $ computeScale (appearanceBox . objectAppearance $ o)
```
Comment (alois9866) [inline src/Visual/Renderer.hs]: Why `scale’` is a lambda while getOffset is not?

File: src/Visual/Renderer.hs [hunk 2]
```diff
@@ -44,3 +42,14 @@ positionPicture camera pic = Scale ratio ratio (Translate x y pic)
...
+                                                    (appearanceActualSize . objectAppearance $ o)
+        getOffset o = ( (positionToVector . fst . appearanceBox . objectAppearance $ o) `plus`
```
Comment (alois9866) [inline src/Visual/Renderer.hs]: It would be better to use `plus` in prefix form IMO.

Comment (dm-gh) [reply src/Visual/Renderer.hs]: I'll leave this here just for the record
capture

File: src/Visual/Renderer.hs (hunk 1 above)
Comment (dm-gh) [reply src/Visual/Renderer.hs]: done

File: src/Visual/Renderer.hs (hunk 2 above)
Comment (dm-gh) [reply src/Visual/Renderer.hs]: done

**Pull request merged**